_config.yml

```diff
@@ -32,14 +32,22 @@ description:
   cn: "建立在CryptoNote的基础上，AEON是一种轻量的隐私电子货币，比如区块链修剪技术的使用。来了解一下这位门罗币的小弟的特点吧"
   fr: "Basée sur les Cryptonotes, AEON est une devise digitale privée de caractère léger comme l’élagage de la blockchain. Venez visiter la petite sœur de Monero!"
 tagline:
-  en: "<strong>AEON</strong><br/>A lightweight digital currency"
-  es: "<strong>AEON</strong><br/>Una moneda digital liviana"
-  eo: "<strong>AEON</strong><br/>Malpeza cifereca monero"
-  de: "<strong>AEON</strong><br/>Eine leichte digitale Währung"
-  pl: "<strong>AEON</strong><br/>Lekka waluta cyfrowa"
-  rs: "<strong>AEON</strong><br/>Легкая цифровая валюта"
-  cn: "<strong>AEON</strong><br/>輕量級數字貨幣"
-  fr: "<strong>AEON</strong><br/>Une monnaie numérique légère"
+  es: "<strong>AEON</strong> is.."
```

Comment:
this is espanol, english missing since editing

Reply:
I will fix that